cypress/e2e/dfi.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { exchanges } from "../fixture/dfi.config";
+
+const viewports = ["iphone-xr", "ipad-2", "macbook-16"];
+
+viewports.forEach((viewport) => {
+  context(`/explore/dfi on ${viewport}`, () => {
+    beforeEach(() => {
+      cy.visit("/explore/dfi");
+      cy.viewport(<Cypress.ViewportPreset>viewport);
+    });
+
+    it("should have Explore DFI section visible and expected text", () => {
+      it("should be visible with expected text", () => {
+        cy.checkElementVisibilityAndText(
+          "section-title-explore-dfi",
+          "EXPLORE THE POWER OF DFI"
+        );
+        cy.checkElementVisibilityAndText(
+          "section-sub-title",
+          "Experience DeFi with DFI"
+        );
+        cy.checkElementVisibilityAndText(
+          "section-desc",
+          "Our native token unlocks the power of decentralized finance, giving you access to a growing ecosystem of cutting-edge DeFi tools and innovative blockchain applications."
+        );
+      });
+    });
+
+    it("should have Start exploring section clickable", () => {
+      if (viewport === "macbook-16") {
+        cy.findByTestId("start-exploring-button").click({
+          scrollBehavior: false,
+        });
+        cy.url().should("include", "#statistics-display-dfi");
+      }
+      cy.checkElementVisibilityAndText(
+        "statistic-title-dfi-minted",
+        "TOTAL DFI MINTED"
+      );
+      cy.checkElementVisibilityAndText(
+        "statistic-title-circulating-supply",
+        "CIRCULATING SUPPLY"
+      );
+    });
+
+    it("should have Building better tomorrow section visible ", () => {
+      cy.checkElementVisibilityAndText(
+        "section-title-building-better-tomorrow",
+        "BUILDING A BETTER TOMORROW WITH DFI"
+      );
+      cy.checkElementVisibilityAndText(
+        "harness-dfi-power-title",
+        "Harness the power of DFI"
+      );
+      cy.checkElementVisibilityAndText(
+        "harness-dfi-power-desc",
+        "A gateway to decentralized applications and tools, offering potential for greater returns, lower fees, and democratic governance."
+      );
+
+      cy.get("article > span.font-bold.mb-4")
```

Comment:
Don't use this. replace it with text id

Reply:
We need to anchor it to a specific element and not tie it to structure (HTML and Style).